pkg/crds/yaml/generated/management.cattle.io_roletemplates.yaml

```diff
@@ -92,7 +92,7 @@ spec:
               type: string
             type: array
           rules:
-            description: Rules holds all the PolicyRules for this RoleTemplate
+            description: Rules holds all the PolicyRules for this RoleTemplate.
```

Comment:
```suggestion
            description: Rules hold all the PolicyRules for this RoleTemplate.
```

Reply:
As noted in other comments, this is a generated file so I can't make changes here. I've made changes in the source file though.